src/controller.rs

```diff
@@ -150,13 +159,31 @@ async fn launch_job(
             backoff_limit: Some(1),
             template: PodTemplateSpec {
                 spec: Some(PodSpec {
+                    volumes: Some(vec![Volume {
+                        name: "credentials".to_string(),
+                        secret: Some(SecretVolumeSource {
+                            secret_name: Some("gar-docker-secret".to_string()),
+                            items: Some(vec![KeyToPath {
+                                key: ".dockerconfigjson".to_string(),
+                                path: "config.json".to_string(),
+                                ..Default::default()
+                            }]),
+                            ..Default::default()
+                        }),
+                        ..Default::default()
+                    }]),
```

Comment:
Currently fighting with permission issues on the `kubecfg` spawned `Job` side. I thought this `credentials` mount would fix it, unfortunately not so far.

```
failed to authorize: failed to fetch oauth token: unexpected status from GET request to <registry> : 401 Unauthorized
```

I'll continue digging into this.

Reply:
Fixed :rocket: